usr/lib/systemd/user/pipewire-rnnoise-switch.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
I feel like there must be a way to set the default source without a service like this.